napari/layers/utils/layer_utils.py

```diff
@@ -809,7 +819,7 @@ def currents(self) -> Dict[str, np.ndarray]:
 
     def set_currents(
         self,
-        currents: Dict[str, np.ndarray],
+        currents: Dict[str, npt.NDArray],
```

Comment:
To clarify, `npt.NDArray` is slightly stronger than `np.ndarray`? Specifically that the `dtype` is a scalar?